src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_module_constants.h

```diff
@@ -151,7 +151,7 @@ const char kFrequencyFraction[] = "frequencyFraction";
 const char kBand[] = "band";
 const char kRdsData[] = "rdsData";
 const char kHdRadioEnable[] = "hdRadioEnable";
-const char kAvailableHDs[] = "availableHDs";
```

Comment:
The parameter availableHDs is only deprecated. These constants should be kept until the parameter is fully removed from the APIs.

Reply:
@JackLivio Thanks. Fixed in be2dd5badfc9fef8734b56fce1a1827f8ac1aee0